docs/UserGuide.md

```diff
@@ -330,15 +333,30 @@ Shows a list of all events saved in Athena's calendar.
 
 Format: `list -e`
 
+#### Sorting displayed events : `sort`
+
+Sorts all currently displayed events in Athena.
+
+Format: `sort -e INDEX`
+
+* Sorts through all events currently on screen and lists them according to the user-requested order
+* Index definitions:
+1. By lexicographical order of the events' descriptions
+2. By lexicographical order of the events' timing
+
+Examples:
+* `sort -e 2` would sort all currently displayed events by their descriptions in alphabetical order.
```

Comment:
This does not seem to follow the general example format of other commands. The format usually goes as 
"`{command}` Sorts all currently ... " starting with a capitalized verb